core/variables.js

```diff
@@ -581,6 +620,19 @@ Blockly.Variables.createVariable_ = function(workspace, id, opt_name,
   return variable;
 };
 
+/**
+ * Helper function to create a button for the flyout window.
```

Comment:
Note that this creates the XML for the button, not the button itself.  That's because the XML then gets passed into the flyout code, which creates the button onscreen.